ios/swift/WaasSdkGo.xcframework/Info.plist

```diff
@@ -16,20 +16,6 @@
 			<key>SupportedPlatform</key>
 			<string>ios</string>
 		</dict>
-		<dict>
-			<key>LibraryIdentifier</key>
-			<string>ios-x86_64-simulator</string>
```

Comment:
- WaasSdkGo as-stated doesn't work on my M2 Mac (OSX 13.3, Xcode 14.2).
- There's some linking error re: having a duplicate target, implying that the binary is not merged correctly as-is.